101-machine-learning-linkedserivce-create/azuredeploy.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "amlWorkspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the AML workspace"
+            }
+        },
+        "linkName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the LinkedService"
+            }
+        },
+        "synapseWorkspaceResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "ResourceId of the Synapse workspace which you want to link to the aml workspace."
+            }
+        },
+        "sparkPools": {
+            "type": "array",
+            "defaultValue": [

```

Comment:
remove the defaultValue since this sample requires an existing one

Reply:
My intention is to show some sample code to customer on how to specify this parameter. Removed and add some extra info in metadata description.